moveit_experimental/jog_arm/CMakeLists.txt

```diff
@@ -1,31 +1,68 @@
-set(JOG_ARM_NAME jog_arm)
+cmake_minimum_required(VERSION 2.8.3)
+project(jog_arm)
```

Comment:
Are we happy naming this `jog_arm` or should this have a `moveit` prefix?

Reply:
I'm definitely agree to name this package `moveit_jog_arm` in order to distinguish it from the existing `jog_arm` package. Actually, I wanted to ask @AndyZe to apply this change after the present PR is merged ;-)
Do you think, it should be part of the present PR instead?

Reply:
Well, I like the shorter name.

`jog_arm` was never released as a debian for Melodic so there won't be a conflict there (and I don't mind removing the Kinetic debian from rosdistro). So don't worry about name conflicts.

Reply:
Made a PR to remove the old version of `jog_arm` from rosdistro here:

https://github.com/ros/rosdistro/pull/21987

Reply:
All packages of the MoveIt universe are prefixed with `moveit_`. This should hold for `jog_arm` as well. That's the "downside" of your decision to migrate the package into MoveIt :wink:

Reply:
I like ``moveit_jog_arm``

The upside is we have really good tools in the moveit repo for code quality and maintenance :-)